<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QMainWindow" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1051</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>用户</string>
     </attribute>
     <widget class="QTableWidget" name="twUser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>831</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bSetpoint">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>30</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置积分</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefresh">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>120</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>任务</string>
     </attribute>
     <widget class="QTableWidget" name="twTask">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>821</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefreshTask">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
